docs/CODEOWNERS

```
@@ -0,0 +1,10 @@
+# See https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+# The CODEOWNERS file itself should have eyes on it.
+docs/CODEOWNERS @akf @arpage
+
+# Benefits-finder tool team 
+web/modules/custom/usagov_benefit_finder @scottqueen-bixal @gchi25
+config/sync/*bears* @scottqueen-bixal @gchi25
+config/sync/*benefit_* @scottqueen-bixal @gchi25
+config/sync/*benefits_finder* @scottqueen-bixal @gchi25
```

```suggestion
config/sync/*benefits_finder* @scottqueen-bixal @gchi25
.github/workflows/*benefits-finder* @scottqueen-bixal @XavierMetichecchia 
```

We could consider adding any associated actions

---

Added!